app/src/ble.c

```diff
@@ -548,6 +548,28 @@ static void zmk_ble_ready(int err) {
     update_advertising();
 }
 
+int zmk_ble_reset_clear_all_bonds() {
+    LOG_WRN("Clearing all existing BLE bond information from the keyboard");
+
+    for (int i = 0; i < 10; i++) {
+        bt_unpair(i, NULL);
```

Comment:
For *splits*, this *won't* work well for users. In particular, if we're invoking this from the keymap, we'll end up unpairing the other half in addition to the host bonds.

I think we need to instead do this inside the loop over the profiles below, and unpair the explicit address found in each profile before deleting the setting, so we don't mess w/ the split bonds when we do this.

And.... when we do this, we need to be careful to actually clear *everything* for the "settings reset Kconfig" setting though, since that *should* be clearing the split bonds too.